Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,23 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
+#ifdef ELB_FULL_GRAPHIC_CONTROLLER
+      lcd_contrast += encoderPosition;
+      lcd_contrast &= 0xFF;
+#else
```

Comment:
Minor syntactical thing. Compiler directives should be part of the cascade too. This is so that naïve text editors can do code folding properly.